Purchase order docs v2.12 (#197)
msupply-foundation/msupply_docs: content/docs/replenishment/purchase orders/index.md

### Foreign Currencies

You can select a foreign currency for the purchase order. This will usually be in your suppliers currency. Click on the dropdown menu and select the currency that you would like to use for the purchase order.

<div class="tip">See the <a href="/docs/introduction/faq/#is-there-support-for-my-currency">currency support question</a> for the list of codes supported</div>

### View or edit the Purchase Order Information Panel

The Information Panel allows you see or to edit information about the Purchase Order. It is divided in multiple sections:

- Pricing
- Other
- Dates

#### How to open and close the Information Panel ?

To open the Information Panel, you can tap on the `More` button, located in the top right corner of the Purchase Order view.

You can close by tapping on the `X Close` button, on the top right corner of the information panel.

![Information Panel](images/po_information_panel.png)

#### Pricing

In the **Pricing** section, you can see pricing information for the Purchase Order.

This includes:

- Subtotal

> - Additional fees (entered in the #XXDetails tab)

Reviewer: ```suggestion
- Additional fees (entered in the <a href="#details">Details</a> tab)
```

- Supplier discount percentage
- Supplier discount amount
- Final cost

#### Other

In the **Other** section, you can:

- Donor name

> - Select a shipping method (if you have this configured in mSupply XX)

Reviewer: ```suggestion
- Select a shipping method (if you have this <a href="https://docs.msupply.org.nz/issuing_goods:issuing_goods_customer_invoice?s%5B%5D=shipping&s%5B%5D=method#shipping_method">configured in mSupply</a>)
```

- Write or edit a comment

<div class="note">

> Configuring donors is done in the mSupply central server. This [documentation page](https://docs.msupply.org.nz/receiving_goods:donors?s[]=donor#adding_or_editing_donors) will tell you how.

Reviewer: ```suggestion
Configuring donors is done in the mSupply central server. This <a href="https://docs.msupply.org.nz/receiving_goods:donors?s[]=donor#adding_or_editing_donors">documentation page</a> will tell you how.
```

the markup syntax does not work within an html element

</div>

#### Dates

In this section, you will see key dates for the Purchase Order.

- Confirmed
- Sent
- Contract signed
- Advance paid

### Purchase Order Status Sequence

The status sequence is located at the bottom left corner of the Purchase Order screen.

Passed statuses are highlighted in blue, next statuses appear in grey.

<figure> 
    <img src="images/po_statussequence1.png" alt="Purchase Order Status Sequence (Sent)" style="width:100%">

>     <figcaption align="center">Status Sequence: current status is <code>New</code>.</figcaption>

Reviewer: ```suggestion
    <figcaption align="center">Status Sequence: current status is <code>Sent</code>.</figcaption>
```

the status doesn't match the image

| **Ready for Sending**  | The purchase order is ready to send to the supplier                                                                           |   cn    |                |                  ✓                  |
| **Sent**               | The purchase order has been sent to the supplier. Purchase order lines will be updated to the `Sent` status                   |   cn    |                |                  ✓                  |
| **Finalised**          | When you confirm that the order has been received. Purchase order lines will be updated to the `Closed` status                |   fn    |                |                                     |

The Ready for Approval status only applies if the Authorise Purchase Orders global preference is turned on. If the preference is off, purchase orders will progress from New status to Ready for Sending.

You'll see that the status bar has only the status values which apply depending on Authorisation preferences.

<figure align="center">
    <img src="images/po_statussequence2.png" alt="Purchase Order Status Sequence (Ready for approval)" style="width:60%"> 

>     <figcaption align="center">Status Sequence: current status is <code>Ready for approval</code>.</figcaption>

Reviewer: ```suggestion
    <figcaption align="center">Status Sequence: current status is <code>Ready for Sending</code>.</figcaption>
```

though the case should be 'Ready for sending'

| **Finalised**          | Finalised          |                                      |                                       |

Purchase orders must proceed through the statuses in the exact order. The status can also go back to a previous step when certain actions are taken. The system will always show you the next available status on the button.

Clicking on the confirm button will show a prompt to proceed, which may have further information.

![](images/po_confirm.png)

## Adding lines to an Purchase Order

To add a line, tap on the `Add Item` button located in the top left corner of your screen.

A new `Add Item` window opens.

![Purchase order: add item button](images/po_add_item.png)

### Select an Item

In the `Add Item` window, you can look up an item by:

- Reading through the list of available items
- or by typing some or all of an item name
- or by typing some or all of an item code

Once your item is highlighted, tap on the name or press `Enter`.

![Add Item list](images/po_add_item_list.png)
![Add Item by name](images/po_add_item_name.png)
![Add Item by code](images/po_add_item_code.png)

> ### Item Details

Reviewer: :nit 
```suggestion
### Item details
```
no need for Caps on the Details

+++
title = "Purchase Orders"
description = "Requesting stock from external suppliers."
date = 2025-09-26
updated = 2025-09-26
draft = false
weight = 44
sort_by = "weight"
template = "docs/page.html"

[extra]
toc = true
top = false
+++

Intro

## Viewing Purchase Orders

### Go to the Purchase Orders Menu

First you need to have purchase orders enabled for your store.

> 1. Go to the `Manage` menu in the navigation panel

Reviewer: Can you link to the store prefs page pls: `/docs/manage/facilities/#store-preferences`

2. Click on `Stores`, then select your store
3. In the `Preferences` section, enable `Use procurement functionality`

![Go to Purchase Orders](images/po_navigation.png)

Now you will be able to access the **Purchase Order** menu:

1. Go to the `Replenishment` menu in the navigation panel

> 2. Click on `Purchase Orders`

Reviewer: having the image where it is makes it look like it is part of the preferences task. I'd suggest moving it down:

```suggestion
Now you will be able to access the **Purchase Order** menu:

![Go to Purchase Orders](images/po_navigation.png)

1. Go to the `Replenishment` menu in the navigation panel
2. Click on `Purchase Orders`

```

Reviewer: I've also resized the image, it's 50% smaller and looks better I think

| Column                 | Description                           |
| :--------------------- | :------------------------------------ |
| **Supplier**           | Name of the Supplier                  |
| **Number**             | Purchase Order Number                 |
| **Created**            | Created date                          |
| **Confirmed**          | Confirmed date                        |
| **Sent**               | Sent date                             |
| **Requested delivery** | Requested delivery date               |
| **Status**             | Current status                        |
| **Target Months**      | Target months supply                  |
| **Lines**              | Number of lines on the purchase order |
| **Comment**            | Purchase order comment                |

2. The list can display a fixed number of purchase orders per page. On the bottom left corner, you can see how many purchase orders are currently displayed on your screen.

3. If you have more purchase orders than the current limit, you can navigate to the other pages by clicking on the page number or using the right of left arrows (bottom right corner).

> 4. You can also select a different number of rows to show per page using the option at the bottom right of the page.

Reviewer: we can remove this bit - we should add a general 'how to use tables' section instead 🙏

Author: Agreed, that would be nice especially with the upcoming table changes!

This section is in many pages currently, so a lot of this duplication will be able to be removed then too 😁

| **Sent**               | Sent               | Finalised                            | Finalised                             |
| **Finalised**          | Finalised          |                                      |                                       |

Purchase orders must proceed through the statuses in the exact order. The status can also go back to a previous step when certain actions are taken. The system will always show you the next available status on the button.

Clicking on the confirm button will show a prompt to proceed, which may have further information.

![](images/po_confirm.png)

## Adding lines to an Purchase Order

To add a line, tap on the `Add Item` button located in the top left corner of your screen.

A new `Add Item` window opens.

![Purchase order: add item button](images/po_add_item.png)

### Select an Item

In the `Add Item` window, you can look up an item by:

- Reading through the list of available items
- or by typing some or all of an item name
- or by typing some or all of an item code

Once your item is highlighted, tap on the name or press `Enter`.

![Add Item list](/images/po_add_item_list.png)
![Add Item by name](/images/po_add_item_name.png)

> ![Add Item by code](/images/po_add_item_code.png)

Reviewer: ```suggestion
![Add Item list](images/po_add_item_list.png)
![Add Item by name](images/po_add_item_name.png)
![Add Item by code](images/po_add_item_code.png)
```
